lib/charms/ovn_charm.py

```diff
@@ -299,6 +300,53 @@ def enable_openstack(self):
         """
         return reactive.is_flag_set('charm.ovn-chassis.enable-openstack')
 
+    @property
+    def additional_dpdk_libraries(self):
+        """A list of additional runtime libraries to be installed for dpdk.
+
+        :returns: list of additional packages to install
+        :rtype: List[str]
+        """
+        packages = []
+        if self.options.enable_dpdk and self.options.dpdk_runtime_libraries:
+            # dpdk_runtime_libraries is a space delimited list of strings.
+            # some options are disabled by passing 'None' so filter out a
+            # specifying of a 'None' value
+            pkgs = list(filter(lambda x: x and x.lower() != 'none',
```

Comment:
This works but I would also test passing 'none' and multiple values in the unit tests.

Reply:
Done in latest push